src/plugins/visualization_ui_components/public/components/field_picker/field_picker.tsx

```diff
@@ -11,7 +11,7 @@ import React, { useRef } from 'react';
 import { i18n } from '@kbn/i18n';
 import useEffectOnce from 'react-use/lib/useEffectOnce';
 import { EuiComboBox, EuiComboBoxProps, EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
-import { FieldIcon } from '@kbn/unified-field-list-plugin/public';
+import { FieldIcon } from '@kbn/unified-field-list/src/components/field_icon';
```

Comment:
I've got a questions for @elastic/kibana-operations 
So this change of import from a plugin to a package is adding 7.9Kb to the Page load bundle of `visualizationUiComponents` . `field_icon` was just moved from a plugin to the package. So I wonder why did this increase so much? thx

Reply:
I did a quick test on bundle size and found out that by enabling the treeshaking at the package level these ~7.9kb can be removed.
What I found is that this is imported into the first page load bundle without threeshaking:

<img width="811" alt="Screenshot 2023-06-12 at 11 19 53" src="https://github.com/elastic/kibana/assets/924948/1119e089-9ca2-4b29-bde1-30bd580af524">

Which is roughly 7.9 kb:

<img width="633" alt="Screenshot 2023-06-12 at 11 18 30" src="https://github.com/elastic/kibana/assets/924948/2804bd39-74f7-4f2f-a960-d3b01e7c4dab">

Adding the `sideEffects: false` flag into the new package makes thing as before:
<img width="647" alt="Screenshot 2023-06-12 at 11 21 51" src="https://github.com/elastic/kibana/assets/924948/8a15489e-6d6a-474a-8ccf-7b72041899b7">

In the past I've managed to trim down quite a bit of various plugins with the same technique [here](https://github.com/elastic/kibana/pull/156669).

Reply:
This is great knowhow 👍! Thank you!